flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -241,7 +241,6 @@ under the License.
 			<groupId>org.apache.hive</groupId>
 			<artifactId>hive-exec</artifactId>
 			<version>${hive.version}</version>
-			<scope>provided</scope>
```

Comment:
What is the implication from removing this?

Reply:
It makes hive-exec a transitive dependency. It won't be packed into hive connector because we no longer build a shaded jar.